src/google/protobuf/io/tokenizer.cc

```diff
@@ -1003,11 +1003,12 @@ bool Tokenizer::ParseInteger(const std::string& text, uint64_t max_value,
 
 double Tokenizer::ParseFloat(const std::string& text) {
   double result;
```

Comment:
let's initialize this to zero just in case we are in a non-debug build.  I would rather have a well specified behavior then UB

Reply:
```suggestion
  double result = 0;
```

Reply:
Done